docs-2.0/nebula-studio/manage-schema/st-ug-crud-index.md

```diff
@@ -37,16 +30,20 @@ To create an index on the **Schema** page, follow these steps:
 
    a. **Index Type**: Choose to create an index for a tag or for an edge type. In this example, **Edge Type** is chosen.
 
-   b. **Name**: Choose a tag name or an edge type name. In this example, **action** is chosen.
+   b. **Name**: Choose a tag name or an edge type name. In this example, **follow** is chosen.
+
+   c. **Index Name**: Specify a name for the new index. In this example, **follow_index** is used.
 
-   c. **Index Name**: Specify a name for the new index. In this example, **action_index** is used.
+   d. **Indexed Properties**: Click **Add**, and then, in the dialog box, choose a property. If necessary, repeat this step to choose more properties. You can drag the properties to sort them. In this example, `degree` are chosen.
 
-   d. **Indexed Properties**: Click **Add**, and then, in the dialog box, choose a property. If necessary, repeat this step to choose more properties. You can drag the properties to sort them. In this example, `actionId` and `label` are chosen.
-   !!! note
+    !!! note
 
-       The order of the indexed properties has an effect on the result of the `LOOKUP` statement. For more information, see [nGQL Manual](https://docs.nebula-graph.io/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/lookup-syntax/#error_code_411 "Click to go to the Nebula Graph website").
+        The order of the indexed properties has an effect on the result of the `LOOKUP` statement. For more information, see [nGQL Manual](../../3.ngql-guide/7.general-query-statements/5.lookup.md "Click to go to the Nebula Graph website").
+
+   e. **Comment**: The remarks of a certain property or the index itself. The maximum length is 256 bytes. By default, there will be no comments on a index, `follow_index` is used.
```

Comment:
"By default, there will be no comments on a index, `follow_index` is used."
Something wrong with the logic of this sentence. Please double-check it.
And it should be "no comments on an index".